src/test_l1_tvSettings.c

```diff
@@ -2096,11 +2095,11 @@ void test_l1_tvSettings_negative_SaveBacklight (void)
 	int videoFormat = 0;
 	int count = 0;
 	bool SupportAvailable = true;
-    char keyValue[UT_KVP_MAX_ELEMENT_SIZE] = { 0 };
+	char keyValue[UT_KVP_MAX_ELEMENT_SIZE] = { 0 };
 
-    videoSource = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoSource/index/0");
-    pqValue = UT_KVP_PROFILE_GET_UINT32("tvSettings/PictureMode/index/0");
-    videoFormat = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoFormat/index/0");
+	videoSource = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoSource/index/0");
```

Comment:
Instead of using direct int, unsigned short int,  char. replace it with the proper data types like int32_t, uint16_t etc.

Reply:
updated

Reply:
```c
tvVideoSrcType_t videoSource;
...
videoSource = (tvVideoSrcType_t ) UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoSource/index/0");
```

Remove all the cases from your code where your constantly casing the variables that should be real types.

Reply:
updated

Reply:
int, unsigned int and many data types are still with default values instead of int32_t, unit16_t etc. Need to be get corrected part of clean up activity